### PR TITLE
Adding learning objectives, questions, key points to 08-filters

### LINKED ISSUE #23: 07-filters needs learning objectives etc
`07-filters.md` front matter needs some:

- [ ] questions
- [ ] learning objectives
- [ ] key points

### PR DESCRIPTION
**Pull request by tobyhodges**
Fixes #23

### CODE REVIEW

File: _episodes/08-filters.md [hunk 1]
```diff
@@ -3,11 +3,17 @@ title: "Working with Filters"
 teaching: 0
 exercises: 0
 questions:
-- "Key question (FIXME)"
+- "How can I specify the format of interpolated variables?"
```
Comment (sstevens2) [inline _episodes/08-filters.md]: Could we reword this one?  From my level of jekyll knowledge, I'm not sure I quite understand the question.

**Comment by sstevens2** [comment]
Added one comment on a question I don't understand and it looks good to me!

File: _episodes/08-filters.md (hunk 1 above)
Comment (tobyhodges) [reply _episodes/08-filters.md]: What about

> How can I control the format of variable values such as dates when inserting them into my pages?

Comment (sstevens2) [reply _episodes/08-filters.md]: I think that sounds great!

**Pull request merged**